uae4all_gp2x_0.7.2a/src/linetoscr.h

```diff
@@ -8,31 +8,55 @@ static __inline__ int LNAME (int spix, int dpix, int stoppos)
     if (bpldualpf) {
 	    // OCS/ECS Dual playfield 
 	    int *lookup = bpldualpfpri ? dblpf_ind2 : dblpf_ind1;
-		
-	    while (dpix < stoppos) {
+		int n = (stoppos-dpix);
+    
+        //center fix mithrendal
+        dpix = (dpix-VISIBLE_LEFT_BORDER)*2 + VISIBLE_LEFT_BORDER;
+	    while (n--) {
 			register unsigned short d = colors_for_drawing.acolors[lookup[pixdata.apixels[spix]]];
 			buf[dpix++] = d;
-			spix += SRC_INC;
+            buf[dpix++] = d;
+            spix ++;
 	    }
 		
```

Comment:
Strange in XCode the indentation looks right. But I discovered that line 17 "buf[dpix++] = d;" indents with 3 tabs whereas line 18 indents with 12 spaces.  So I guess Xcode uses the width of 4 spaces for one tab character whereas the github diff uses the width of 8 spaces for a tab character.

Any idea how to get this right? Is Xcode able to convert indentation spaces to indentation tabs automatically ?